Remove Travis and Codacy Badge Table

kulloveth opened the pull request:
### Description
Removed travis and codacy badges table

Fixes #1155 

### Type of Change:
Removed travis and codacy badges table

**Code/Quality Assurance Only**




### How Has This Been Tested?
Checked the ReadMe and the table has been removed

kulloveth committed:
remove travis and codacy badge table

kulloveth committed:
fix merge conflict

isabelcosta reviewed (changes requested):
@kulloveth thank you for contributing!
Could you remove the files unrelated to this issue from git? These would be any file that is not README.

kulloveth committed:
remove unrelated files

kulloveth commented:
> @kulloveth thank you for contributing!
> Could you remove the files unrelated to this issue from git? These would be any file that is not README.

I have removed them

isabelcosta reviewed (changes requested):
Thank you for addressing the comments! Can you please revert the changes done on `.gitignore` and `buildSrc` folders, they show up as deleted on "Files Changed" tab on this pull request.

kulloveth committed:
added back gitignore and dependencies.kt

kulloveth commented:
> Thank you for addressing the comments! Can you please revert the changes done on `.gitignore` and `buildSrc` folders, they show up as deleted on "Files Changed" tab on this pull request.

done

kulloveth commented:
seems like I have to also add gradle-wrapper.properties back, the workflow seems to failing because of it @isabelcosta

gaurivn commented:
> seems like I have to also add gradle-wrapper.properties back, the workflow seems to failing because of it @isabelcosta

@kulloveth, how is it going, pls update

kulloveth commented:
hey @gaurivn yeah Isabel requested I remove files that were not related to the changes but then I see the build is failing because the gradle wrapper is missing. was asking if I should add it back

isabelcosta commented on gradle/wrapper/gradle-wrapper.properties:
```diff
@@ -1,6 +0,0 @@
-#Tue Jul 24 16:33:21 IST 2018
```
This file was removed, I think this is making the build fail. Could you undo this removal?

isabelcosta replied:
the change was still there, so I removed it for you :) Now you have to do `git pull` on your end :)

isabelcosta commented on buildSrc/src/main/kotlin/Dependencies.kt:
```diff
@@ -8,6 +8,7 @@ object Versions {
     const val versionCode = 1
     const val versionName = "1"
     const val gradleBuildTool = "4.2.1"
+    const val dataBinding = "3.2.0-alpha11"
```
can you also remove this line, as its not related to the changes needed :)

kulloveth replied:
> can you also remove this line, as its not related to the changes needed :)

Done

kulloveth committed:
made requested changes

isabelcosta commented on gradle/wrapper/gradle-wrapper.properties:
```diff
@@ -1,6 +1,6 @@
 distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```
```suggestion
zipStoreBase=GRADLE_USER_HOME
zipStorePath=wrapper/dists
```

isabelcosta committed:
Update gradle/wrapper/gradle-wrapper.properties

isabelcosta commented on gradle/wrapper/gradle-wrapper.properties:
```diff
@@ -1,4 +1,4 @@
-#Tue Jul 24 16:33:21 IST 2018
+#Sun Sep 05 15:10:40 WAT 2021
```
```suggestion
#Tue Jul 24 16:33:21 IST 2018
```

isabelcosta committed:
Update gradle/wrapper/gradle-wrapper.properties

isabelcosta reviewed (approved):
Thank you for contributing @kulloveth 🙌🏾

kulloveth commented:
> Thank you for contributing @kulloveth 🙌🏾

Missed that one, thank you

gaurivn reviewed (approved):
Looks good to me :+1:

vj-codes reviewed (approved):
@kulloveth congratulations on your first PR to the repository! 
LGTM🎉
Quick tip : create a new separate branch on your fork repo and then commit the changes when contributing in the future. It is always a good open source practice and mentioned in the community guidelines too:)

vj-codes commented:
Adding label ready to merge since it's a documentation change

isabelcosta commented:
> @kulloveth congratulations on your first PR to the repository!
> LGTM🎉
> Quick tip : create a new separate branch on your fork repo and then commit the changes when contributing in the future. It is always a good open source practice and mentioned in the community guidelines too:)

Great tip @vj-codes ! 🤗
